### ISSUE TITLE
variant enum

### ISSUE THREAD

**Issue by Astrapolis-peasant** [rocklabs-io/ic-py#48]
for variant type like
```
 variant {
   a;
   b;
 };
```
use `canister.xxx({"a": None})` returns `AttributeError: 'NoneType' object has no attribute 'buildTypeTable'`

**Comment by Myse1f** [comment]
Currently, variant is represented by:
```
None: [] (Empty list),
Some: [xxx] (list with one item)
```

Thus you need to use `canister.xxx({"a": []})`

**Comment by Myse1f** [comment]
> Currently, variant is represented by:
> 
> ```
> None: [] (Empty list),
> Some: [xxx] (list with one item)
> ```
> 
> Thus you need to use `canister.xxx({"a": []})`

Sry, I got it wrong. `Option` is used in this way.

I will look into this issue.

**Comment by Myse1f** [comment]
Thanks for the report. Try to fix it in #49.

You can switch to that branch and use `canister.xxx({"a": None})` to test whether the result is correct.
